src/routers/options.ts

```diff
@@ -3,13 +3,14 @@ import { default as express } from 'express';
 import type * as db from '../db';
 import { isLoggedIn } from '../middleware/isLoggedIn';
 import { getOption } from '../services/options';
-import { saveComment, getCommentsForOption } from '../services/comments';
+import { saveComment, getCommentsForOption, deleteComment } from '../services/comments';
 
 const router = express.Router();
 
 export function optionsRouter({ dbPool }: { dbPool: PostgresJsDatabase<typeof db> }) {
   router.get('/:optionId', isLoggedIn(), getOption(dbPool));
   router.get('/:optionId/comments', isLoggedIn(), getCommentsForOption(dbPool));
   router.post('/:optionId/comments', isLoggedIn(), saveComment(dbPool));
+  router.delete('/:optionId/comments', isLoggedIn(), deleteComment(dbPool));
```

Comment:
ideally we add /comments/:commentId to the delete route so it is understood that we will not delete all comments and then we can use that param to delete it

Reply:
ahh okey makes sense :)